### PR TITLE
Cropping Images Dynamically

### PR DESCRIPTION
**Pull request by Pranjalmishra30**
Code to crop images by selecting a region of the image. 
Also can save the cropped image. Concepts of MouseEvents are used in this script

Original Issue : https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93#issuecomment-689375050

### CODE REVIEW

**Comment by Pranjalmishra30** [comment]
@disha2sinha @ankitdobhal please review my PR.

File: Image-Processing/ImageCropper/crop.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import cv2
+refpt = [] #List of refrence points
+
+def select_roi (event,x,y,flags,param):
+    global refpt #Global refrences
+
+    if event == cv2.EVENT_LBUTTONDOWN: # When the left mouse button is cliked
+        refpt = [(x,y)]
```
Comment (disha2sinha) [inline Image-Processing/ImageCropper/crop.py]: Put space after comma

File: Image-Processing/ImageCropper/crop.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    elif event == cv2.EVENT_LBUTTONUP: # When the left mouse button is released
+        refpt.append((x,y)) # recording the last coordinates
```
Comment (disha2sinha) [inline Image-Processing/ImageCropper/crop.py]: Put Space after comma

File: Image-Processing/ImageCropper/crop.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+while True:
+    cv2.imshow("frame",img_main)
+    var = cv2.waitKey(0)
+
+    # Select a region , then press c to crop that portion
+
+    if var == ord('c'): # Crop selected images
+        
+        if len(refpt)==2:
```
Comment (disha2sinha) [inline Image-Processing/ImageCropper/crop.py]: Put space around ==

File: Image-Processing/ImageCropper/crop.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            roi = clone2[refpt[0][1]:refpt[1][1], refpt[0][0]:refpt[1][0]] # [x1:x2 , y1:y2]
```
Comment (disha2sinha) [inline Image-Processing/ImageCropper/crop.py]: Put space around :

**Review by disha2sinha — CHANGES_REQUESTED** [review]
Put space after comma and around operators wherever applicable. Rest all is fine

**Comment by Pranjalmishra30** [comment]
Updated the file with the suggested changes , please review.

**Review by powerexploit — COMMENTED** [review]
Nice @Pranjalmishra30 .

**Review by powerexploit — APPROVED** [review]

**Pull request merged**